docroot/sites/all/modules/features/ilr_permissions/ilr_permissions.features.user_permission.inc

```diff
@@ -2882,6 +2912,17 @@ function ilr_permissions_user_default_permissions() {
     'module' => 'paragraphs_bundle_permissions',
   );
 
+  // Exported permission: 'view paragraph content side_by_side_text'.
+  $permissions['view paragraph content side_by_side_text'] = array(
+    'name' => 'view paragraph content side_by_side_text',
+    'roles' => array(
+      'administrator' => 'administrator',
+      'anonymous user' => 'anonymous user',
+      'contributor' => 'contributor',
```

Comment:
Per our discussion in sprint planning, a new story has been created to handle permissions for paragraphs, https://www.pivotaltracker.com/story/show/120110333. For future reference, the view permissions should always be anonymous and registered users, which covers all the roles. The current setting would leave out registered users who do not have either the contributor or administrator role.